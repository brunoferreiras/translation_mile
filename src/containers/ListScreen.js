import React, { Component } from 'react';
import { View } from 'react-native';
import { Header } from 'react-native-elements';

class ListScreen extends Component {
  render() {
    return (
      <View>
        <Header
          statusBarProps={{}}
          leftComponent={{ icon: 'menu', color: '#fff' }}
          centerComponent={{
            text: 'Translation Mile',
            style: { color: '#fff', fontSize: 20 }
          }}
          rightComponent={{ icon: 'home', color: '#fff' }}
          outerContainerStyles={{ backgroundColor: '#8CB13E' }}
          innerContainerStyles={{ justifyContent: 'space-between' }}
        />
      </View>
    );
  }
}

export default ListScreen;
