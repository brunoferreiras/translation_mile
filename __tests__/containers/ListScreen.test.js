import React from 'react';
import renderer from 'react-test-renderer';
import ListScreen from '../../src/containers/ListScreen';

describe('ListScreen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ListScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
