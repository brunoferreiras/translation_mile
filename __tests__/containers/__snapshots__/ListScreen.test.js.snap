// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListScreen renders correctly 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#476DC5",
          "borderBottomColor": "#f2f2f2",
          "borderBottomWidth": 1,
          "height": 70,
          "padding": 15,
        },
        undefined,
        Object {
          "backgroundColor": "#8CB13E",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "flex-end",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          },
          Object {
            "justifyContent": "space-between",
          },
        ]
      }
    >
      <View
        style={
          Array [
            false,
            false,
            false,
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
        underlayColor="white"
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#fff",
                "fontSize": 24,
              },
              Array [
                Object {
                  "backgroundColor": "transparent",
                },
                undefined,
              ],
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
      <Text
        numberOfLines={1}
        style={
          Array [
            Object {},
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "color": "#fff",
              "fontSize": 20,
            },
          ]
        }
      >
        Translation Mile
      </Text>
      <View
        style={
          Array [
            false,
            false,
            false,
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
        underlayColor="white"
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#fff",
                "fontSize": 24,
              },
              Array [
                Object {
                  "backgroundColor": "transparent",
                },
                undefined,
              ],
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;
